src/java.desktop/windows/native/libawt/windows/awt_DnDDS.cpp

```diff
@@ -39,10 +39,14 @@ void * operator new(size_t size) {return operator new(size, "stl", 1);}
 #pragma pop_macro("bad_alloc")
 //"bad_alloc" is undefined from here
 
-#include <awt.h>
 #include <shlobj.h>
 
+// Don't want to pull in the redefined allocation functions
```

Comment:
Maybe this needs to be clarified a bit:

```suggestion
// These files must be included before awt.h, since the latter redefines malloc 
// to Do_Not_Use_Malloc, etc, and that will break these files.
```

Also, please mark conversations as resolved in the GitHub UI once you think they are addressed. That makes it much easier for reviewers to understand what parts of the reviews that have not been addressed.